### ISSUE TITLE
Куда обращаться в техподдержку, если я разрабатываю клиент для tinkoff API в личных целях как pet-project?

### ISSUE THREAD

**Issue by IvanIvanov1985** [Tinkoff/investAPI#471]
Заранее прошу прощения, если вопрос глупый, но вроде в доках нашёл только про поддержку публичных сервисов, и по поиску в issues не нашёл про тех поддержку.

Я так понял что здесь публично тоже будет неправильно скидывать requests id или что-то подобное. Или в случае личного проекта тоже можно писать на al.a.volkov@tinkoff.ru?

**Comment by nonamegithub** [comment]
Да, непростой случай: для [релоканта](https://github.com/Tinkoff/investAPI/issues/443), да вдобавок [разработчика](https://github.com/Tinkoff/investAPI/issues/470#issuecomment-1663377447), вступают в силу определенные ограничения, накладываемые законодательством страны, предоставившей убежище. А вдруг в ней запрещено разрабатывать pet-проекты на основе публичных сервисов? Плюс самый сложный момент - если страна, в которой вы окажетесь/оказались, находится в списке [недружественных государств](https://ru.wikipedia.org/wiki/%D0%A1%D0%BF%D0%B8%D1%81%D0%BE%D0%BA_%D0%BD%D0%B5%D0%B4%D1%80%D1%83%D0%B6%D0%B5%D1%81%D1%82%D0%B2%D0%B5%D0%BD%D0%BD%D1%8B%D1%85_%D0%B3%D0%BE%D1%81%D1%83%D0%B4%D0%B0%D1%80%D1%81%D1%82%D0%B2_(%D0%A0%D0%BE%D1%81%D1%81%D0%B8%D1%8F)). Тут у вас не остается никакого морального права на использование [InvestAPI](https://github.com/Tinkoff/investAPI/), так как Банк Tinkoff [попал под санкции](https://www.tinkoff.ru/blog/whats-new/sanctions/) государства-[сюзерена](https://ru.wikipedia.org/wiki/%D0%A1%D1%8E%D0%B7%D0%B5%D1%80%D0%B5%D0%BD) недружественных государств. Знаю, что среди пользователей репозитория есть релоканты, находящиеся в Армении, но судя по их постоянному недовольству, у них тоже не все гладко.
